dependencies/bullseye/hammer_cli_foreman_host_reports/ruby-tests.rake

```diff
@@ -0,0 +1,6 @@
+require 'gem2deb/rake/testtask'
```

Comment:
can you try dropping this file and see what happens? I think that's the one that executes the tests for us.